x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/manage_crawls_popover/manage_crawls_popover.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiButton, EuiContextMenu, EuiPopover } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { CrawlerDomain } from '../../types';
+
+// import { AutomaticCrawlScheduler } from './automatic_crawl_scheduler';
```

Comment:
Are you intentionally keeping this line (and the other commented code)?

Reply:
Yep, that will be done in a follow-up PR